content/reference-rippled.md

```diff
@@ -8345,6 +8368,11 @@ The `info` object may have some arrangement of the following fields:
 | load.job\_types | Array | *Admin only* Information about the rate of different types of jobs the server is doing and how much time it spends on each. |
 | load.threads | Number | *Admin only* The number of threads in the server's main job pool. |
 | load\_factor | Number | The load factor the server is currently enforcing, as a multiplier on the base transaction cost. The load factor is determined by the highest of the individual server's load factor, cluster's load factor, and the overall network's load factor. **Note:** This `load_factor` is calculated as the ratio of the `load_factor` and the `load_base` that are reported by the [`server_state` command](#server-state) |
+| load\_factor\_local | Number | (May be omitted) Current multiplier to the [transaction cost][] based on load to this server. |
+| load\_factor\_net | Number | (May be omitted) Current multiplier to the [transaction cost][] being used by the rest of the network (estimated). |
+| load\_factor\_cluster | Number | (May be omitted) Current multiplier to the [transaction cost][] based on load to servers in [this cluster](tutorial-rippled-setup.html#clustering). |
+| load\_factor\_fee\_escalation | Number | (May be omitted) The current multiplier to the [transaction cost][] that a transaction must pay to get into the open ledger. _(New in [version 0.32.0][])_ |
+| load\_factor\_fee\_queue | Number | (May be omitted) The current multiplier to the [transaction cost][] that a transaction must pay to get into the queue, if the queue is full. _(New in [version 0.32.0][])_ |
```

Comment:
I find the phrase "The current multiplier to the [transaction cost][] that a transaction must pay" a little confusing. Is the value in this field what I need to pay? Or do I need to multiply my transaction cost by this? (Again, I think this might be implied, but would probably be better to make it totally clear.) Also, what are "fee levels" (don't see this in the diff, but I see it here: http://mduo13.github.io/ripple-dev-portal/reference-rippled.html#server-state).